content/docs/command-reference/dag.md

```diff
@@ -69,22 +71,31 @@ $ dvc exp show ...
 
 ## Options
 
-- `--full` - show full DAG that the `target` stage belongs to, instead of
-  showing only its ancestors.
+- `-h`, `--help` - prints the usage/help message, and exit.
 
-- `--dot` - show DAG in
-  [DOT](<https://en.wikipedia.org/wiki/DOT_(graph_description_language)>)
-  format. It can be passed to third party visualization utilities.
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
```

Comment:
-hqv should be at the end. Not sure how why they were moved up.

Reply:
Ah, probably because of https://github.com/iterative/dvc.org/pull/3454#discussion_r862525475 -- I didn't mean -hqv though 🙂

I'll fix it...